mrp_client.py

```diff
@@ -19,15 +19,16 @@ def __init__(self, parser, args):
             exit(1)
 
     def parse(self):
-        if self.args.subcommand == 'ip':
+        if self.args.subcommand == 'net':
             print(self.getNetworkInfo(self.args.action, self.args.machine,
                                       self.args.interface))
         else:
             self.parser.print_help()
 
-    def getNetworkInfo(self, request, machine_name, interface_name='eth1'):
+    def getNetworkInfo(self, request, machine_name, interface_name):
         try:
-            machine = MachineControl(self.urlhandler, machine_name)
+            machine = MachineControl(self.urlhandler, machine_name,
+                                     interface_name)
```

Comment:
This is the right place to implement the concatenation.

Reply:
What concatenation ?